test/dummy/db/schema.rb

```diff
@@ -218,6 +218,8 @@
     t.boolean "merge_queue_enabled", default: false, null: false
     t.datetime "last_deployed_at"
     t.integer "repository_id", null: false
+    t.datetime "archived_since"
```

Comment:
Had you considered `archived_on` to keep with rails conventions for date/time stamps?

Reply:
I considered it but maintained consistency with fields like `locked_since` instead. I can do whatever @byroot prefersn

Reply:
I considered it but maintained consistency with fields like `locked_since` instead. I can do whatever @byroot prefers.

Reply:
"archived_since" is fine.